tests/CONTACTS/test_27038.py

```diff
@@ -37,8 +37,9 @@ def test_run(self):
 
         # Cancel the login
         self.marionette.switch_to_frame()
-        cancel = self.UTILS.element.getElements(DOM.Contacts.import_cancel_login, "Cancel icon")
-        cancel[-1].tap()
+        cancel = self.UTILS.element.getElement(('xpath', '//h1[contains(text(), "Google")]/..'), "Cancel icon")
```

Comment:
There was several matches for the generic locator ('css selector', gaia-header[action=close]) so I had to come up with this :S